### ISSUE TITLE
PROJ6/GDAL 2.5 SRS test failures

### ISSUE THREAD

**Issue by hobu** [PDAL/PDAL#2383]
We will need to tweak the tests a little bit to catch up to WKT output changes from GDAL for the new gdalbarn.com combo.

```
[==========] Running 15 tests from 1 test case.
[----------] Global test environment set-up.
[----------] 15 tests from SpatialReferenceTest
[ RUN      ] SpatialReferenceTest.test_ctor
[       OK ] SpatialReferenceTest.test_ctor (0 ms)
[ RUN      ] SpatialReferenceTest.test_proj4_roundtrip
[       OK ] SpatialReferenceTest.test_proj4_roundtrip (1 ms)
[ RUN      ] SpatialReferenceTest.test_userstring_roundtrip
../test/unit/SpatialReferenceTest.cpp:109: Failure
Value of: wkt
  Actual: "GEOGCS[\"WGS 84\",DATUM[\"WGS_1984\",SPHEROID[\"WGS 84\",6378137,298.257223563,AUTHORITY[\"EPSG\",\"7030\"]],AUTHORITY[\"EPSG\",\"6326\"]],PRIMEM[\"Greenwich\",0,AUTHORITY[\"EPSG\",\"8901\"]],UNIT[\"degree\",0.0174532925199433,AUTHORITY[\"EPSG\",\"9122\"]],AUTHORITY[\"EPSG\",\"4326\"]]"
Expected: ret_wkt
Which is: "GEOGCS[\"WGS 84\",DATUM[\"WGS_1984\",SPHEROID[\"WGS 84\",6378137,298.257223563,AUTHORITY[\"EPSG\",\"7030\"]],AUTHORITY[\"EPSG\",\"6326\"]],PRIMEM[\"Greenwich\",0,AUTHORITY[\"EPSG\",\"8901\"]],UNIT[\"degree\",0.0174532925199433,AUTHORITY[\"EPSG\",\"9122\"]],AXIS[\"Latitude\",NORTH],AXIS[\"Longitude\",EAST],AUTHORITY[\"EPSG\",\"4326\"]]"
[  FAILED  ] SpatialReferenceTest.test_userstring_roundtrip (7 ms)
[ RUN      ] SpatialReferenceTest.test_get_utmzone
unknown file: Failure
C++ exception with description "Minimum zone is -38' and maximum zone is '-37'. They do not match because they cross a zone boundary" thrown in the test body.
[  FAILED  ] SpatialReferenceTest.test_get_utmzone (195 ms)
[ RUN      ] SpatialReferenceTest.calcZone
[       OK ] SpatialReferenceTest.calcZone (0 ms)
[ RUN      ] SpatialReferenceTest.test_read_srs
[       OK ] SpatialReferenceTest.test_read_srs (10 ms)
[ RUN      ] SpatialReferenceTest.test_writing_vlr
../test/unit/SpatialReferenceTest.cpp:241: Failure
Value of: reference_wkt
  Actual: "GEOGCS[\"WGS 84\",DATUM[\"WGS_1984\",SPHEROID[\"WGS 84\",6378137,298.257223563,AUTHORITY[\"EPSG\",\"7030\"]],AUTHORITY[\"EPSG\",\"6326\"]],PRIMEM[\"Greenwich\",0,AUTHORITY[\"EPSG\",\"8901\"]],UNIT[\"degree\",0.0174532925199433,AUTHORITY[\"EPSG\",\"9122\"]],AUTHORITY[\"EPSG\",\"4326\"]]"
Expected: wkt
Which is: "GEOGCS[\"WGS 84\",DATUM[\"WGS_1984\",SPHEROID[\"WGS 84\",6378137,298.257223563,AUTHORITY[\"EPSG\",\"7030\"]],AUTHORITY[\"EPSG\",\"6326\"]],PRIMEM[\"Greenwich\",0,AUTHORITY[\"EPSG\",\"8901\"]],UNIT[\"degree\",0.0174532925199433,AUTHORITY[\"EPSG\",\"9122\"]],AXIS[\"Latitude\",NORTH],AXIS[\"Longitude\",EAST],AUTHORITY[\"EPSG\",\"4326\"]]"
../test/unit/SpatialReferenceTest.cpp:281: Failure
Value of: reference_wkt
  Actual: "GEOGCS[\"WGS 84\",DATUM[\"WGS_1984\",SPHEROID[\"WGS 84\",6378137,298.257223563,AUTHORITY[\"EPSG\",\"7030\"]],AUTHORITY[\"EPSG\",\"6326\"]],PRIMEM[\"Greenwich\",0,AUTHORITY[\"EPSG\",\"8901\"]],UNIT[\"degree\",0.0174532925199433,AUTHORITY[\"EPSG\",\"9122\"]],AUTHORITY[\"EPSG\",\"4326\"]]"
Expected: wkt
Which is: "GEOGCS[\"WGS 84\",DATUM[\"WGS_1984\",SPHEROID[\"WGS 84\",6378137,298.257223563,AUTHORITY[\"EPSG\",\"7030\"]],AUTHORITY[\"EPSG\",\"6326\"]],PRIMEM[\"Greenwich\",0,AUTHORITY[\"EPSG\",\"8901\"]],UNIT[\"degree\",0.0174532925199433,AUTHORITY[\"EPSG\",\"9122\"]],AXIS[\"Latitude\",NORTH],AXIS[\"Longitude\",EAST],AUTHORITY[\"EPSG\",\"4326\"]]"
[  FAILED  ] SpatialReferenceTest.test_writing_vlr (11 ms)
[ RUN      ] SpatialReferenceTest.test_io
[       OK ] SpatialReferenceTest.test_io (1 ms)
[ RUN      ] SpatialReferenceTest.test_vertical_and_horizontal
../test/unit/SpatialReferenceTest.cpp:316: Failure
Value of: horizontal
  Actual: "GEOGCS[\"WGS 84\",DATUM[\"WGS_1984\",SPHEROID[\"WGS 84\",6378137,298.257223563,AUTHORITY[\"EPSG\",\"7030\"]],AUTHORITY[\"EPSG\",\"6326\"]],PRIMEM[\"Greenwich\",0,AUTHORITY[\"EPSG\",\"8901\"]],UNIT[\"degree\",0.0174532925199433,AUTHORITY[\"EPSG\",\"9122\"]],AXIS[\"Latitude\",NORTH],AXIS[\"Longitude\",EAST],AUTHORITY[\"EPSG\",\"4326\"]]"
Expected: horiz
Which is: "GEOGCS[\"WGS 84\",DATUM[\"WGS_1984\",SPHEROID[\"WGS 84\",6378137,298.257223563,AUTHORITY[\"EPSG\",\"7030\"]],AUTHORITY[\"EPSG\",\"6326\"]],PRIMEM[\"Greenwich\",0,AUTHORITY[\"EPSG\",\"8901\"]],UNIT[\"degree\",0.0174532925199433,AUTHORITY[\"EPSG\",\"9122\"]],AUTHORITY[\"EPSG\",\"4326\"]]"
../test/unit/SpatialReferenceTest.cpp:317: Failure
Value of: vertical
  Actual: "VERT_CS[\"NAVD88 height\",VERT_DATUM[\"North American Vertical Datum 1988\",2005,EXTENSION[\"PROJ4_GRIDS\",\"g2003conus.gtx\"],AUTHORITY[\"EPSG\",\"5103\"]],UNIT[\"metre\",1,AUTHORITY[\"EPSG\",\"9001\"]],AXIS[\"Up\",UP],AUTHORITY[\"EPSG\",\"5703\"]]"
Expected: vert
Which is: "VERT_CS[\"NAVD88 height\",VERT_DATUM[\"North American Vertical Datum 1988\",2005,AUTHORITY[\"EPSG\",\"5103\"],EXTENSION[\"PROJ4_GRIDS\",\"g2003conus.gtx\"]],UNIT[\"metre\",1,AUTHORITY[\"EPSG\",\"9001\"]],AXIS[\"Up\",UP],AUTHORITY[\"EPSG\",\"5703\"]]"
[  FAILED  ] SpatialReferenceTest.test_vertical_and_horizontal (0 ms)
[ RUN      ] SpatialReferenceTest.readerOptions
(readers.las Error) Geotiff directory contains key 0 with short entry and more than one value.
(readers.las Error) Geotiff directory contains key 0 with short entry and more than one value.
(readers.las Error) Geotiff directory contains key 0 with short entry and more than one value.
(readers.las Error) Geotiff directory contains key 0 with short entry and more than one value.
(readers.las Error) Geotiff directory contains key 0 with short entry and more than one value.
(readers.las Error) Geotiff directory contains key 0 with short entry and more than one value.
(readers.las Error) Geotiff directory contains key 0 with short entry and more than one value.
(readers.las Error) Geotiff directory contains key 0 with short entry and more than one value.
(readers.las Error) Geotiff directory contains key 0 with short entry and more than one value.
(readers.las Error) Geotiff directory contains key 0 with short entry and more than one value.
(readers.las Error) Geotiff directory contains key 0 with short entry and more than one value.
(readers.las Error) Geotiff directory contains key 0 with short entry and more than one value.
(readers.las Error) Geotiff directory contains key 0 with short entry and more than one value.
(readers.las Error) Geotiff directory contains key 0 with short entry and more than one value.
(readers.las Error) Geotiff directory contains key 0 with short entry and more than one value.
(readers.las Error) Geotiff directory contains key 0 with short entry and more than one value.
[       OK ] SpatialReferenceTest.readerOptions (85 ms)
[ RUN      ] SpatialReferenceTest.merge
(readers.las Error) Geotiff directory contains key 0 with short entry and more than one value.
(readers.las Error) Geotiff directory contains key 0 with short entry and more than one value.
(readers.las Error) Geotiff directory contains key 0 with short entry and more than one value.
(readers.las Error) Geotiff directory contains key 0 with short entry and more than one value.
../test/unit/Support.cpp:328: Failure
Value of: v2->getFieldAs<double>(Id::X, idx)
  Actual: -83.427599999999998
Expected: v1->getFieldAs<double>(Id::X, idx)
Which is: 39.012599999999999
Index = 0
[  FAILED  ] SpatialReferenceTest.merge (298 ms)
[ RUN      ] SpatialReferenceTest.test_bounds
../test/unit/SpatialReferenceTest.cpp:454: Failure
Value of: -83.42759776f
  Actual: -83.427597
Expected: static_cast<float>(b2.minx)
Which is: 39.0126
../test/unit/SpatialReferenceTest.cpp:455: Failure
Value of: 39.01259905f
  Actual: 39.0126
Expected: static_cast<float>(b2.miny)
Which is: -83.427597
../test/unit/SpatialReferenceTest.cpp:456: Failure
Value of: -83.427551f
  Actual: -83.427551
Expected: static_cast<float>(b2.maxx)
Which is: 39.012619
../test/unit/SpatialReferenceTest.cpp:457: Failure
Value of: 39.01261687f
  Actual: 39.012615
Expected: static_cast<float>(b2.maxy)
Which is: -83.427551
[  FAILED  ] SpatialReferenceTest.test_bounds (2 ms)
[ RUN      ] SpatialReferenceTest.identifyEPSG
[       OK ] SpatialReferenceTest.identifyEPSG (1 ms)
[ RUN      ] SpatialReferenceTest.issue_1989
[       OK ] SpatialReferenceTest.issue_1989 (32 ms)
[ RUN      ] SpatialReferenceTest.set_srs
[       OK ] SpatialReferenceTest.set_srs (11 ms)
[----------] 15 tests from SpatialReferenceTest (654 ms total)

[----------] Global test environment tear-down
[==========] 15 tests from 1 test case ran. (655 ms total)
[  PASSED  ] 9 tests.
[  FAILED  ] 6 tests, listed below:
[  FAILED  ] SpatialReferenceTest.test_userstring_roundtrip
[  FAILED  ] SpatialReferenceTest.test_get_utmzone
[  FAILED  ] SpatialReferenceTest.test_writing_vlr
[  FAILED  ] SpatialReferenceTest.test_vertical_and_horizontal
[  FAILED  ] SpatialReferenceTest.merge
[  FAILED  ] SpatialReferenceTest.test_bounds
```

**Comment by hobu** [comment]
```
[ RUN      ] FerryFilterTest.test_ferry_copy_json
(Error) GDAL failure (1) SetCompoundCS() fails, vertical component is not VERT_CS.
(filters.reprojection Error) GDAL failure (1) PROJ: proj_create: unrecognized format / unknown name
(filters.reprojection Error) GDAL failure (1) Cannot instanciate target_crs
(filters.reprojection Error) GDAL failure (6) Cannot find coordinate operations from `EPSG:2993' to `'
unknown file: Failure
C++ exception with description "filters.reprojection: Could not construct coordinate transformation object in createTransform" thrown in the test body.
```

**Comment by abellgithub** [comment]
Fixed with #2548
